query/src/main/java/org/apache/kylin/query/relnode/OLAPContext.java

```diff
@@ -337,7 +340,14 @@ private String transferDateTimeColumnToMillis(CompareTupleFilter compFilter, Str
         }
 
         if (column.getType().isDateTimeFamily()){
-            value = String.valueOf(DateFormat.stringToMillis(value));
+            logger.debug("value changed before: {}", value);
+            if(column.getType().isDate()) {
+                // It seems the dynamic parameter has been changed to the date integer value
+                value = String.valueOf(Long.parseLong(value) * 86400000L);
+            } else {
+                value = String.valueOf(DateFormat.stringToMillis(value));
+            }
+            logger.debug("value changed after: {}", value);
```

Comment:
Could you please combine two lines into one line, like : `Column value changed from {} to {}.`